src/index.tsx

```diff
@@ -402,7 +408,7 @@ export class App extends React.Component<{}, AppState> {
             </>
           )}
 
-          {(hasText(elements) || isTextToolSelected) && (
+          {(hasText(elements) || isTextToolSelected || isEditingText) && (
```

Comment:
Whenever isTextToolSelected is `true`, we show the panel, so here we need to decide whether we keep isEditingText or isTextToolSelected cause both might show the panel anyway

Reply:
Not sure what you mean. `isEditingText` is when the wysiwyg contenteditable is active. `isTextToolSelected` is when you just select the tool.